src/main/java/ca/intelliware/ihtsdo/mlds/search/AffiliateSearchResult.java

```diff
@@ -0,0 +1,38 @@
+package ca.intelliware.ihtsdo.mlds.search;
+
+import ca.intelliware.ihtsdo.mlds.domain.Affiliate;
+
+import java.util.Collection;
+
+/**
+ * Supports displaying search result information in the browser.
+ */
+public class AffiliateSearchResult {
+	private Collection<Affiliate> affiliates; // affiliates found with a certain search (paged)
+	private long totalResults; // total number of results for the search
+	private int totalPages; // total number of pages for the search
```

Comment:
Lovely, great use of a collection object rather than a collection of objects which was previously lacking the total